content/applications/inventory_and_mrp/inventory/barcode/operations/receipts_deliveries.rst

```diff
@@ -0,0 +1,183 @@
+:guilabel:`pencil icon` can be clicked to open a new screen to edit that product line.
+
+From this screen, the product that's being received is listed. Under the product name, the
+:guilabel:`Quantity` line can be edited. Either change the `0` in the line to the desired quantity,
+or click the :guilabel:`/# Units` button (in this case, :guilabel:`/10 Units`) to automatically fill
+the quantity ordered from the :abbr:`PO (purchase order)`.
+
+.. image:: receipts_deliveries/receipts-deliveries-product-line-editor.png
+   :align: center
+   :alt: Product line editor for individual transfer in Barcode app.
+
+Additionally, the :guilabel:`+1` and :guilabel:`-1` buttons can be clicked to add or subtract
+quantity of the product, and the :guilabel:`number keys` can be used to add quantity, as well.
+
+Below the :guilabel:`number keys` is the :guilabel:`location` line, which reads `WH/Stock` by
+default, unless another *location* is listed on the product itself. Click this line to reveal a
+drop-down menu of additional locations to choose from.
+
+Once ready, click :guilabel:`Confirm` to confirm the changes made to the product line.
+
+Then, from the overview page with all receipts to process within that transfer (**WH/IN/000XX**),
+click the :guilabel:`+#` button on the product line for the products being received, and click
+:guilabel:`Validate`. The receipt has now been processed, and the :guilabel:`Barcode app` can be
+closed out.
+
+.. image:: receipts_deliveries/receipts-deliveries-validate-transfer.png
+   :align: center
+   :alt: Overview of receipts in transfer to validate.
+
+Scan barcodes for delivery orders
+=================================
+
+To process warehouse deliveries for outgoing products, there first needs to be a sales order (SO)
+created, and a delivery operation to process.
+
+To create a :abbr:`SO (sales order)`, navigate to the :menuselection:`Sales app --> Create` to
+create a new quotation. From the blank quotation form, click the drop-down menu next to the
+:guilabel:`Customer` field to add a customer. Then, on the :guilabel:`Product` line under the
+:guilabel:`Order Lines` tab, click :guilabel:`Add a product`, and select the desired product(s) to
+add to the quotation.
+
+Once ready, click :guilabel:`Save` at the top of the form, and click :guilabel:`Confirm Order` to
+confirm the quotation to a :abbr:`SO (sales order)`.
+
+.. image:: receipts_deliveries/receipts-deliveries-sales-order.png
+   :align: center
+   :alt: Completed sales order for barcode product.
+
+To process and scan barcodes for warehouse deliveries, navigate to the :menuselection:`Barcode app`.
+Once inside the :guilabel:`Barcode app`, a :guilabel:`Barcode Scanning` screen displaying different
+options is presented. To process deliveries, click on the :guilabel:`Operations` button at the
+bottom of the screen. This navigates to an :menuselection:`Operations` overview page.
```

Comment:
if :menuselection:`Operations` is talking about the page itself, then we should use guilabel! If this is for navigational steps, like Barcode > Operations > Receipts, we'd use menuselection.